src/app/shared/components/users-list/users-list.component.ts

```diff
@@ -37,9 +37,10 @@ export class UsersListComponent implements OnInit, AfterViewInit {
   @Input() isEditable: boolean = false;
 
   @Input() displayedColumns: string[] = ['pib', 'email', 'phone', 'place', 'role', 'status', 'actions'];
-
+  @Input() isEdit: boolean = false;
```

Comment:
in this case, a typo is code smell, when for primitive types of variables, specify both the data type and the value. Because isEdit by default is false, it is obviously a boolean; The same thing when you do value: string = "hello", etc.